addons/account_peppol/wizard/account_move_send.py

```diff
@@ -115,9 +115,9 @@ def _call_web_service_after_invoice_pdf_render(self, invoices_data):
 
         invoices_data_peppol = {}
         for invoice, invoice_data in invoices_data.items():
-            if invoice_data.get('ubl_cii_xml_attachment_values'):
-                xml_file = invoice_data['ubl_cii_xml_attachment_values']['raw']
-                filename = invoice_data['ubl_cii_xml_attachment_values']['name']
+            if invoice_data.get('xml_attachment_values'):
+                xml_file = invoice_data['xml_attachment_values']['raw']
+                filename = invoice_data['xml_attachment_values']['name']
```

Comment:
Useless renaming. We want to know it comes from ubl or cii so keep the original name

Reply:
Why does `account.move.send` need to be aware of it?
Shouldn't the edi model take care of all the specificities and keep the sending flow general and agnostic from what it's sending?